mage_ai/data_preparation/templates/data_exporters/streaming/azure_data_lake.yaml

```diff
@@ -0,0 +1,6 @@
+connector_type: azure_data_lake
+table_uri: abfs://container@storagename.dfs.core.windows.net/
+account_name: account_name
+access_key: access_key
+mode: append
+file_type: delta
```

Comment:
nit: add newline to end of the file

Reply:
fixed on 23cc8513f148e9eb7e07fd3d283fe472ea62eae5